css/components/navigation.css

```
@@ -0,0 +1,90 @@
+.nav-bar {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--primary-color);
+  padding: 1rem;
+}
+.nav-section-items i {
+  font-size: 1.5rem;
+  color: var(--white-color);
```

```suggestion
  color: var(--white-color);
```
This property is used several times. Instead, use it once in parent, it will get inherited by default.